development/docker-compose.base.yml

```
@@ -21,8 +21,9 @@ services:
         condition: "service_started"
       db:
         condition: "service_healthy"
-    <<: *nautobot-build
-    <<: *nautobot-base
+    <<:
+      - *nautobot-base
+      - *nautobot-build
```

```suggestion
    <<: =[*nautobot-base, *nautobot-build]
```
Suggestion to follow what has come before.